pages/main.php

```diff
@@ -44,11 +48,11 @@
 if ($func == 'setstatus') {
   $sql = rex_sql::factory();
   $status = (rex_request('oldstatus', 'int') + 1) % 2;
-  $query = "SELECT `term`, `active` FROM rex_multiglossar WHERE `pid` ='" . addslashes($oid) . "' ";
+  $query = "SELECT `term`, `active` FROM " . rex::getTable('multiglossar') . " WHERE `pid` ='" . addslashes($oid) . "' ";
   $sql->setQuery($query);
   if ($sql->getRows() == 1) {
     $term = $sql->getValue('term');
-    $query = "UPDATE  rex_multiglossar  SET `active` = '$status' WHERE `pid` ='" . addslashes($oid) . "' ";
+    $query = "UPDATE  " . rex::getTable('multiglossar') . "  SET `active` = '$status' WHERE `pid` ='" . addslashes($oid) . "' ";
```

Comment:
Addslashes ist hier definitiv nicht richtig. Zum escapen von parametern in raw sql gibt ws in rex_sql eine escape methode.